doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1646,47 +1704,81 @@ For example, to configure the parameters `archive-path` and `archive-max-filesiz
 
 11. `archive-default-encryption-key` - Default archive encryption key (in `archive-encryption-keys-dir`) if there is no grouping or a encryption key for a member is not present. If not defined, the primary GPG encryption key is used.
 
-#### 11.1.4 Archive encryption
 
-Archive files can be encrypted (`archive-encryption-enabled = true`) using GnuPG (OpenPGP compatible, RFC 4880). Please see e.g. [RFC 4880](https://www.ietf.org/rfc/rfc4880.txt) and [GnuPG](https://gnupg.org/) for more infomation about OpenPGP and GnuPG.
+#### 11.1.6 Archive Files
+
+Archive files (ZIP containers) are located in the directory specified by the configuration parameter `archive-path`. File names are in the format `mlog[-grouping]-X-Y-Z.zip[.gpg]`, where X is the timestamp (UTC time in the format `YYYYMMDDHHmmss`) of the first message log record, Y is the timestamp of the last message log record (records are processed in chronological order) and Z is 10 characters long alphanumeric random. If grouping is enabled, [-grouping] is a (possibly truncated and filename safe) member identifier. If encryption is enabled, the `[.gpg]` suffix is added.
+
+The most basic example of an archive file name when the encryption and grouping are switched off:
+
+    mlog-20210901100858-20210901100905-5wOUQnXK4G.zip
+
+When the archive encryption switched on:
+
+    mlog-20210901101923-20210901101926-pXAZkBjnAC.zip.gpg
+
+Switching on archive grouping by member produces the following:
+
+    mlog-INSTANCE_CLASS_CODE-20210901102251-20210901102254-yoA2z7EAkg.zip.gpg
+
+Finally, switching to archive grouping by subsystem gives:
+
+    mlog-INSTANCE_CLASS_CODE_CONSUMERSUBSYSTEM-20210901102521-20210901102524-m1s2v5qZRc.zip.gpg
+    mlog-INSTANCE_CLASS_CODE_PROVIDERSUBSYSTEM-20210901102521-20210901102524-7bZ0LoXNbu.zip.gpg
+
+
+#### 11.1.7 Archive Encryption and Grouping
+
+The archive files can be encrypted (`archive-encryption-enabled = true`) using GnuPG (OpenPGP compatible, RFC 4880). Please see e.g. [RFC 4880](https://www.ietf.org/rfc/rfc4880.txt) and [GnuPG](https://gnupg.org/) for more infomation about OpenPGP and GnuPG.
```

Comment:
I'd add a minor clarification here:

> The archive files can be encrypted (`archive-encryption-enabled = true`) using GnuPG (OpenPGP compatible, RFC 4880). The encryption is enabled/disabled on a security server level - it's not possible to enable/disable it for specific subsystems only. Please see e.g. [RFC 4880](https://www.ietf.org/rfc/rfc4880.txt) and [GnuPG](https://gnupg.org/) for more infomation about OpenPGP and GnuPG.

Reply:
Updated in https://github.com/nordic-institute/X-Road/pull/1038/commits/1bfba1fffbb487b9838d8f805a07d8b394903578